### PR TITLE
Fixing errors

### PR DESCRIPTION
**Pull request by eGzIT**
Fixing small errors in html and css. Added appropriate size of fonts etc etc.

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1013/200021.273:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (d3usss) [inline debug.log:1]: what is that file? Please remove it if not needed

File: index.html [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container-fluid">
+            <div class="row navbar">
+                <div class="col-sm-12">
+                    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+                        <img src="img/logo.jpg" alt="logo">
```
Comment (d3usss) [inline index.html:20]: logo shoud be also a link

File: index.html [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+                <div class="container">
+                    <div class="row about-top">
+                        <div class="col-sm-12">
+                            <h1><span class="square">&nbsp;</span>O szkole</h1>
+                            <p>czyli jak pracuję?</p>
+                        </div>
+                    </div>
+                    <div class="row about">
+                        <div class="about__study">
+                            <div class="col-sm-12">
```
Comment (d3usss) [inline index.html:61]: I blelive col-sm shoud be before about__study class div or mayby you not need two divs but one?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+                                <img src="img/study.svg" class="img-fluid rounded mx-auto d-block">
+                                <h1 class="text-center">CZAS TRWANIA ZAJĘĆ</h1>
+                                <p>Czas trwania jednej lekcji będzie zależeć od twoich preferencji. Możesz wybrać 60, 45, a nawet 30 minut, jeśli nie możesz wygospodarować więcej czasu. Niektórzy wolą spotykać się na 1,5 godziny bez żadnej przerwy, a inni kilka razy w tygodniu. Wybór należy do ciebie. Na pewno ustalimy razem dogodny termin.</p>
+                            </div>
+                        </div>
+                        <div class="about__class">
```
Comment (d3usss) [inline index.html:67]: the same as prev comment

File: index.html [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+                            <a href="" target="_blank">UMOWA WSPÓŁPRACY</a>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        </div>
+
+        <!--Your Teacher-->
+        <div class="container-fluid">
+            <section>
```
Comment (d3usss) [inline index.html:98]: this secitos, do sometching? It have no col class

File: index.html [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+                </div>
+                <div class="row">
+                    <div class="price-list">
+                        <div class="col-sm-12">
+                            <img src="img/teaching.svg" class="img-fluid rounded mx-auto d-block">
+                            <h1 class="text-center">Zajęcia indywidualne</h1>
+                            <p>Jeśli zależy ci na jak naszybszych efektach, indywidalnym podejściu albo dostosowaniu zajęć wyłącznie pod twój grafik wolnego czasu to zajęcia indywidualne będą idealną opcją.</p>
+                            <h2 class="text-center">CZAS I CENA</h2>
+                            <div class="price-list__price">
+                                <p class="text-center">60 <span class="min">min</span> - 50 zł</p>
```
Comment (d3usss) [inline index.html:132]: do you need this class for every p? It's not workign if you add it to parent conteiner?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+                        <h4 class="text-center">Skontaktuj się ze mną</h4>
+                        <p>Aby ze mną skontaktować, proszę skorzyrać z formularza mailowego lub wybrac ć jedną z podanych opcji danych kontaktowych i skontaktow
+                            ać się bezpośrednio. (Jeśli nie odbieram, prawdopodbbnie w tym czasie pracuję, więc  napisz mi proszę sms, a na pewno oddzownię :)</p>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col-sm-12 form">
+                        <form action="...">
+                            <input type="text" name="nazwa" value="imię i nazwisko" size="50vh">
```
Comment (d3usss) [inline index.html:168]: plase remove size from html

File: index.html [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+                            <input type="text" name="nazwa" value="adres e-mail" size="50vh">
+                            <input type="text" name="nazwa" value="temat wiadomości" size="50vh">
+                            <input type="text" name="nazwa" value="treść wiadomości" size="50vh"></br>
+
+                            <button type="sumbit">Wyślij wiadomość</button>
+                        </form>
+                    </div>
+
+                    <div class="col-sm-12 data">
+                        <b>Dane kontaktowe:</b>
```
Comment (d3usss) [inline index.html:178]: I think one of h for example h4 or h5 will be better

File: style.css [hunk 9]
```diff
@@ -0,0 +1,329 @@
+* {
```
Comment (d3usss) [inline style.css:1]: you don't need it to be comminted its should be used only for debuging

File: style.css [hunk 10]
```diff
@@ -0,0 +1,329 @@
...
+    font-family: 'Roboto', sans-serif;
+    text-decoration: none;
+    font-weight: 500;
+    padding-top: 2rem;   
+    font-size: 4.8rem;
+    margin: 0;
+    text-align: center;
+}
+
+.about-top .col-sm-12 p{
```
Comment (d3usss) [inline style.css:77]: is this .col-sm-12 class is needed in selector?

File: style.css [hunk 11]
```diff
@@ -0,0 +1,329 @@
...
+.about__class .col-sm-12 img{
+    width: 50%;
+    height: 50%;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+}
+
+/*Effort*/
+.about__effort .col-sm-12 p{
+    font-family: 'Roboto', sans-serif;
```
Comment (d3usss) [inline style.css:133]: i bleive you don't need moust of this styles bocuse you definied genral p and h1 styles on the begnin of file

Comment (d3usss) [reply style.css:133]: please remove all duplicated styles that are not specyfic and different that styles form top of file

File: debug.log (hunk 1 above)
Comment (eGzIT) [reply debug.log:1]: Done.

File: index.html (hunk 2 above)
Comment (eGzIT) [reply index.html:20]: Done.

File: style.css (hunk 11 above)
Comment (eGzIT) [reply style.css:133]: Done.

File: style.css (hunk 10 above)
Comment (eGzIT) [reply style.css:77]: Yes, because later I want to do other settings for md.

File: style.css (hunk 9 above)
Comment (eGzIT) [reply style.css:1]: Okay.

File: index.html (hunk 8 above)
Comment (eGzIT) [reply index.html:178]: Yes, corrected.

File: index.html (hunk 7 above)
Comment (eGzIT) [reply index.html:168]: Done.

File: index.html (hunk 6 above)
Comment (eGzIT) [reply index.html:132]: Yes, otherwise it does not work.

File: index.html (hunk 5 above)
Comment (eGzIT) [reply index.html:98]: Could you check it again? This section has its own content.

File: index.html (hunk 3 above)
Comment (eGzIT) [reply index.html:61]: Otherwise It doesn't work.

File: index.html (hunk 4 above)
Comment (eGzIT) [reply index.html:67]: Same like before.

File: index.html (hunk 3 above)
Comment (d3usss) [reply index.html:61]: Sometching is no, yes :)

File: index.html (hunk 6 above)
Comment (d3usss) [reply index.html:132]: check again please

**Pull request merged**